.gitignore

```diff
@@ -81,3 +81,14 @@ custom-wallet-config.nix
 # wallet web API benchmarking results
 wallet-new/bench/results/*.csv
 wallet-new/bench/results/*.txt
+
+# cardano-state-* for wallet data
+cardano-state-*
```

Comment:
our wiki page: https://github.com/input-output-hk/internal-documentation/wiki/Exchange-requirements

Happy to change it, but we should update our docs to not use that path as well if I do

Reply:
Let's at least change the following line from `state-wallet-mainnet` to `state-*` since they'll be much more common going forward.

After this is merged the internal wiki page should refer to this document instead to avoid duplication/divergence. We can keep this ignore line anyway.